### PR TITLE
Add some image and code examples about TextBox

### PR DESCRIPTION
**Pull request by mcraiha**
Because they are easy to show

### CODE REVIEW

File: wwwroot/docs/controls/textbox.md [hunk 1]
```diff
@@ -1,3 +1,82 @@
...
+		<TextBox PasswordChar="#" />
+	</StackPanel>
+</Window>
+```
+produces following output with **Windows 10** when text is type to input  
+![Password TextBox](images/textbox_password.png)
+
+## TextBox with watermark
+
+With Avalonia and WPF text suggestion that textbox shows is called watermark, but e.g. HTML5 calls it *placeholder*
```
Comment (grokys) [inline wwwroot/docs/controls/textbox.md]: This is a little hard to understand, and what's more I don't think WPF has this feature. Might something like

```suggestion
Avalonia can show a "watermark" in a `TextBox`, which is a piece of text that is displayed when the `TextBox` is empty (in HTML5 this is called a *placeholder*)
```

be better?

File: wwwroot/docs/controls/textbox.md [hunk 2]
```diff
@@ -1,3 +1,82 @@
...
+        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+        mc:Ignorable="d" d:DesignWidth="800" d:DesignHeight="450"
+        x:Class="AvaloniaAppTemplate.MainWindow"
+        Title="AvaloniaAppTemplate">
+	<Grid Margin="10">
+		<TextBox AcceptsReturn="True" TextWrapping="Wrap" />
+	</Grid>
+</Window>
+```
+produces following output with **Windows 10** when text is type to input
```
Comment (grokys) [inline wwwroot/docs/controls/textbox.md]:
```suggestion
produces the following output with **Windows 10** when text is input  
```

File: wwwroot/docs/controls/textbox.md [hunk 3]
```diff
@@ -1,3 +1,82 @@
...
+        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+        mc:Ignorable="d" d:DesignWidth="800" d:DesignHeight="450"
+        x:Class="AvaloniaAppTemplate.MainWindow"
+        Title="AvaloniaAppTemplate">
+	<StackPanel Margin="10">
+		<TextBox Watermark="Street address" />
+	</StackPanel>
+</Window>
+```
+produces following output with **Windows 10**
```
Comment (grokys) [inline wwwroot/docs/controls/textbox.md]:
```suggestion
produces the following output with **Windows 10**  
```

File: wwwroot/docs/controls/textbox.md [hunk 4]
```diff
@@ -1,3 +1,82 @@
...
+        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+        mc:Ignorable="d" d:DesignWidth="800" d:DesignHeight="450"
+        x:Class="AvaloniaAppTemplate.MainWindow"
+        Title="AvaloniaAppTemplate">
+	<StackPanel Margin="10">
+		<TextBox PasswordChar="#" />
+	</StackPanel>
+</Window>
+```
+produces following output with **Windows 10** when text is type to input
```
Comment (grokys) [inline wwwroot/docs/controls/textbox.md]:
```suggestion
produces the following output with **Windows 10** when text is input  
```

File: wwwroot/docs/controls/textbox.md [hunk 5]
```diff
@@ -1,3 +1,82 @@
...
+        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+        mc:Ignorable="d" d:DesignWidth="800" d:DesignHeight="450"
+        x:Class="AvaloniaAppTemplate.MainWindow"
+        Title="AvaloniaAppTemplate">
+	<StackPanel Margin="10">
+		<TextBox />
+	</StackPanel>
+</Window>
+```
+produces following output with **Windows 10**
```
Comment (grokys) [inline wwwroot/docs/controls/textbox.md]:
```suggestion
produces the following output in **Windows 10**  
```

File: wwwroot/docs/controls/textbox.md [hunk 6]
```diff
@@ -1,3 +1,82 @@
 Title: Textbox
 ---
-TODO
\ No newline at end of file
+The `TextBox` is editable text field where user can input text.
```
Comment (grokys) [inline wwwroot/docs/controls/textbox.md]:
```suggestion
The `TextBox` control is an editable text field where a user can input text.
```

File: wwwroot/docs/controls/textbox.md (hunk 1 above)
Comment (mcraiha) [reply wwwroot/docs/controls/textbox.md]: Yeah, sorry I confused plain WPF to MahApps WPF. I used your suggested text in second commit.

File: wwwroot/docs/controls/textbox.md (hunk 2 above)
Comment (mcraiha) [reply wwwroot/docs/controls/textbox.md]: Changed in second commit.

File: wwwroot/docs/controls/textbox.md (hunk 3 above)
Comment (mcraiha) [reply wwwroot/docs/controls/textbox.md]: Changed in second commit.

File: wwwroot/docs/controls/textbox.md (hunk 4 above)
Comment (mcraiha) [reply wwwroot/docs/controls/textbox.md]: Changed in second commit.

File: wwwroot/docs/controls/textbox.md (hunk 5 above)
Comment (mcraiha) [reply wwwroot/docs/controls/textbox.md]: Changed in second commit.

File: wwwroot/docs/controls/textbox.md (hunk 6 above)
Comment (mcraiha) [reply wwwroot/docs/controls/textbox.md]: Changed in second commit.

**Pull request merged**